libs/pika/execution/include/pika/execution/algorithms/require_started.hpp

```diff
@@ -453,7 +453,22 @@ namespace pika {
             {
                 return detail::partial_algorithm<require_started_t>{};
             }
-        } require_started{};
+        };
+
+        /// \brief Diagnose if a sender has not been started and terminates on destruction. It
+        /// forwards the values of the predecessor sender.
+        ///
+        /// Sender adaptor that takes any sender and returns a new sender that sends the same values
+        /// as the predecessor sender.
+        ///
+        /// The destructor is by default terminating if the sender has not been connected or if the
```

Comment:
```suggestion
        /// The destructor terminates if the sender has not been connected or if the
```
I would not mention the "default" since the throwing version isn't something we can do with `stdexec`.